01/1C.hector_hist_co2.Rmd

```diff
@@ -0,0 +1,101 @@
+---
+title: "Hector [CO2] vs. Observations"
+output: html_document
+author: "Peter Scully"
+date: "2024-05-31"
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## Getting the data
+
+First, let's import the relevant libraries and setup our directory location
+```{r import_and_loc}
```

Comment:
When you knit the Rmarkdown there is lots of messages related to installing packages, could you please silence those messages?